infer/annotations/Makefile

```diff
@@ -1,4 +1,4 @@
-CWD = $(shell pwd)
+CWD = $(shell printf "%q\n" "$(shell pwd)")
```

Comment:
this needs `SHELL := /bin/bash`. Could you check that all the Makefiles that use `printf "%q"` set `SHELL` to `/bin/bash`?